src/Models/Hooks/useRuntimeSceneData.ts

```diff
@@ -1,7 +1,9 @@
 import { useEffect, useState } from 'react';
+import { hasBadge } from '../../Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ConditionCalloutUtility';
```

Comment:
this seems bad. Why do we need to reference soething that is nested so deep inside a different component?

Reply:
Can we pull it out to a utility or something instead?